### PR TITLE
Update Introduction.md

### PR DESCRIPTION
**Pull request by venkanavaneetham**
KC, updated intro page with viewer details, thanks.

### CODE REVIEW

File: User Guide/Introduction.md [hunk 1]
```diff
@@ -16,16 +16,25 @@ There are several roles within the ADEX application.
   - **Browse all topics** published by publisher from **other organisations** based on the visibility **"Only my team"**
   - **Publish** and **Subscribe** to topics
   - **Manage teams** such as Create new teams, Edit team details, Add team members, Edit role of team members and Leave team
+  - **Join team** in any organisation, and first time user role will be **Viewer** for the team.
   
 - **Publisher**	
 
   - Browse topics and verify topic details
   - Create new team, Create projects and topics
+  - Join a team in any org as **Viewer**
```
Comment (lohkianchai) [inline User Guide/Introduction.md]: "org" -> "organisation"

File: User Guide/Introduction.md [hunk 2]
```diff
@@ -16,16 +16,25 @@ There are several roles within the ADEX application.
...
   - **Publish** and **Subscribe** to topics
   
 - **Subscriber**
 
   - Browse topics 
   - Create new team, Create only **Subscribe type** projects
+  - Join a team in any org as **Viewer**
```
Comment (lohkianchai) [inline User Guide/Introduction.md]: "org" -> "organisation"

Comment (lohkianchai) [reply User Guide/Introduction.md]: Create new team, Join a team in any organisation.

File: User Guide/Introduction.md (hunk 1 above)
Comment (lohkianchai) [reply User Guide/Introduction.md]: Create new team, Join a team in any organisation.

File: User Guide/Introduction.md [hunk 3]
```diff
@@ -16,16 +16,25 @@ There are several roles within the ADEX application.
   - **Browse all topics** published by publisher from **other organisations** based on the visibility **"Only my team"**
   - **Publish** and **Subscribe** to topics
   - **Manage teams** such as Create new teams, Edit team details, Add team members, Edit role of team members and Leave team
+  - **Join team** in any organisation, and first time user role will be **Viewer** for the team.
   
 - **Publisher**	
 
   - Browse topics and verify topic details
```
Comment (lohkianchai) [inline User Guide/Introduction.md]: Browse topics and update topic details.

File: User Guide/Introduction.md (hunk 2 above)
Comment (venkanavaneetham) [reply User Guide/Introduction.md]: ok KC

File: User Guide/Introduction.md (hunk 1 above)
Comment (venkanavaneetham) [reply User Guide/Introduction.md]: ok KC

**Review by lohkianchai — APPROVED** [review]
ok

**Pull request merged**